src/Fhir.Anonymizer.R4.Core/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Fhir.Anonymizer.Core
+{
+    internal static partial class Constants
+    {
+        // InstanceType constants
+        internal const string supportedVersion = "R4";
```

Comment:
The recommended naming and capitalization convention is to use PascalCasing for constants